src/key_file.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015-2016, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use libc;
+use ffi;
+use translate::*;
+use std;
+use std::ptr;
+use std::path;
+use error::Error;
+use auto::KeyFileFlags;
+
+use KeyFile;
+
+impl KeyFile {
+
+    pub fn error_quark() -> ffi::GQuark {
+        unsafe {
+            ffi::g_key_file_error_quark()
+        }
+    }
```

Comment:
No. The problem is that gir generate code that doesn't build:
````
src/auto/key_file.rs:6:5: 6:10 error: unresolved import `Quark`. There is no `Quark` in the crate root [E0432]
src/auto/key_file.rs:6 use Quark;
                           ^~~~~
src/auto/key_file.rs:6:5: 6:10 help: run `rustc --explain E0432` to see a detailed explanation
````

Fixing gir should help getting rid of that manual wrapper.